app/models/Users.scala

```diff
@@ -60,6 +60,10 @@ object Users {
     findSingleUser { user => user.email === email }
   }
 
+  def findById(id: Int): Option[User] = {
+    findSingleUser { user => user.id === id }
+  }
```

Comment:
(AFAIK) This IS the built-in method for it. Slick uses the standard Scala idioms for managing collections, and converts those to SQL. So 'filter' becomes 'where', 'take' becomes 'limit', etc.
It's kind of cool but feels like you are causing bad queries...